### PR TITLE
Add number proccess locked by an other one

### PR DESCRIPTION
**Pull request by Esysteme**

### CODE REVIEW

**Comment by CharlesJUDITH** [comment]
Can you add at least a description to describe the change? Thx

File: mysql.py [hunk 1]
```diff
@@ -596,6 +604,11 @@ def read_callback():
 	innodb_log_bytes_written = fetch_innodb_os_log_bytes_written(conn)
 	dispatch_value('innodb', 'os_log_bytes_written', innodb_log_bytes_written['COUNT'], 'counter')
 
+
+	meta_data_lock = fetch_mariadb_lock(conn)
+	dispatch_value('mariadb', 'lock', meta_data_lock['nb_lock'], 'gauge')
```
Comment (CharlesJUDITH) [inline mysql.py]: why not using something like information_schema as this is coming from the information_schema table.

**Comment by CharlesJUDITH** [comment]
You should also mention that this new metric is available only when using the metadata_lock plugin: https://mariadb.com/kb/en/information-schema-metadata_lock_info-table/

**Review by CharlesJUDITH — CHANGES_REQUESTED** [review]

**Comment by Esysteme** [comment]
updated index to innodb

**Review by khodin — APPROVED** [review]

**Pull request merged**